book/forms.rst

```diff
@@ -1021,6 +1021,29 @@ to the ``form()`` or the ``form_start()`` helper:
     a PUT, PATCH or DELETE request. Read the cookbook chapter
     ":doc:`/cookbook/routing/method_parameters`" for more information.
 
+.. note::
+
+  Now that the ``createForm()`` method requires the FQCN, you can not longer pass in ``__construct()`` arguments. To get 
```

Comment:
Ah, I get what you're saying now! So, should this example be included in previous symfony version documentation too to ensure developers avoid the constructor argument implementation and *try* to enforce the `$options`  implementation?